scripts/skill_auditor/metrics_extractor.py

```
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+"""Deterministic metrics extraction from SKILL.md files."""
+
+import re
+from pathlib import Path
+from typing import Dict, Any
```

Ruff (0.14.5)

`typing.Dict` is deprecated, use `dict` instead

(UP035)